upgrading/automated_upgrades.adoc

```diff
@@ -20,300 +20,155 @@ endif::[]
-(e.g., 3.10.z), run the following playbook:
-
+.. To upgrade to the latest z-stream version of a minor release, such as 3.11.z,
+run:
++
 ----
 # ansible-playbook \
     -i </path/to/inventory/file> \
     playbooks/byo/openshift-cluster/upgrades/<version>/upgrade.yml
 ----
 
-After rebooting, continue to xref:verifying-the-upgrade[Verifying the Upgrade].
+. After all master and node upgrades have completed, reboot all hosts.
+
+. xref:verifying-the-upgrade[Verify the upgrade].
 endif::[]
 
 ifdef::openshift-enterprise[]
 [[preparing-for-an-automated-upgrade]]
-== Preparing for an Automated Upgrade
-
-[IMPORTANT]
-====
-Before upgrading your cluster to {product-title} 3.10, the cluster must be
-already upgraded to the
-link:https://docs.openshift.com/container-platform/3.9/release_notes/ocp_3_9_release_notes.html#ocp-37-asynchronous-errata-updates[latest asynchronous release of version 3.9]. If your cluster is at a version earlier
-than 3.9, you must first upgrade incrementally. For example, upgrade from 3.6 to
-3.7, then 3.7 to 3.9 (the 3.8 version was link:https://docs.openshift.com/container-platform/3.9/release_notes/ocp_3_9_release_notes.html#ocp-39-about-this-release[skipped]).
-====
-
-[NOTE]
-====
-Before attempting the upgrade, follow the guidance in
-xref:../day_two_guide/environment_health_checks.adoc#day-two-environment-health-checks[Environment
-health checks] to verify the cluster's health. This will confirm that nodes are
-in the *Ready* state, running the expected starting version, and will ensure
-that there are no diagnostic errors or warnings.
-====
+== Preparing for an upgrade
 
-To prepare for an automated upgrade:
+After you satisfy the prerequisites, prepare for an automated upgrade:
 
-. Pull the latest subscription data from RHSM:
+. Pull the latest subscription data from Red Hat Subscription Manager:
 +
 ----
 # subscription-manager refresh
 ----
 
-. If you are upgrading from {product-title} 3.9 to 3.10:
+. If you are upgrading from {product-title} 3.10 to 3.11:
 
-.. Back up the following files that would be required should you later need to
-downgrade back to {product-title} 3.9:
+.. Back up the files that you need if you must downgrade to {product-title}
+3.10:
 
 ... On master hosts, back up the following files:
 +
 ----
-/usr/lib/systemd/system/atomic-openshift-master-api.service
-/usr/lib/systemd/system/atomic-openshift-master-controllers.service
-/etc/sysconfig/atomic-openshift-master-api
-/etc/sysconfig/atomic-openshift-master-controllers
 /etc/origin/master/master-config.yaml
 /etc/origin/master/scheduler.json
 ----
-+
-The *systemd* files are removed during the upgrade due to the changes to the
-control plane static pod architecture.
 
-... On node hosts (including masters, which have the node component on them), back
-up the following files:
+... On node hosts, including masters, back up the following files:
 +
 ----
 /usr/lib/systemd/system/atomic-openshift-*.service
 /etc/origin/node/node-config.yaml
```

Comment:
There is not /usr/lib/systemd/system/atomic-openshift-*.service in v3.10.

Reply:
@kalexand-rh 
Seems this comment did not update in the pr.

Reply:
@jiajliu, "/usr/lib/systemd/system/atomic-openshift-*.service" in not in the file in master.